set.mm

```diff
@@ -427517,18 +427526,18 @@ it can be understood as claiming either uniqueness ( ` E* ` ) or unique
        </li>
 
        <li><p><b>Is-a-set.</b>
-       "` A e. _V ` " should be read "Class ` A ` is a set (i.e. exists)."
+       " ` A e. _V ` " should be read "Class ` A ` is a set (i.e., exists)."
        This is a convention based on Definition 2.9 of [Quine] p. 19.
        See ~ df-v and ~ isset .
        However, instead of using ` I e. _V ` in the antecedent of a theorem for
        some variable ` I ` , we now prefer to use ` I e. V ` (or another
-       variable if ` V ` is not available) to make it more general. That way we
-       can often avoid needing extra uses of ~ elex and ~ syl in the common
-       case where ` I ` is already a member of something.
-       For hypotheses ($e statement) of theorems (mostly in inference form),
-       however, ` |- A e. _V ` is used rather than ` |- A e. V ` (e.g.,
-       ~ difexi ). This is because ` A e. _V ` is almost always satisfied using
-       an existence theorem stating "... ` e. _V `", and a hard-coded ` _V ` in
+       variable if ` V ` is not available) to make it more general.  That way
+       we can often avoid extra uses of ~ elex and ~ syl in the common case
+       where ` I ` is already a member of something.  For hypotheses
+       ($e statement) of theorems (mostly in inference form), however,
+       ` |- A e. _V ` is used rather than ` |- A e. V ` (e.g., ~ difexi ).
+       This is because ` A e. _V ` is almost always satisfied using an
+       existence theorem stating " ` ... e. _V ` ", and a hard-coded ` _V ` in
```

Comment:
Hm, the "symbol" ` ... ` does not mean "ellipsis", but is a "notation for a contiguous finite set of integers". Therefore, I am not sure to include it in the Metamath expression.

There are already cases in set.mm, however, which use ` ... ` as ellipsis in Methamath expressions, e.g., in the header comment of "Class abstractions":

>  1.  The axiomatic statement ~ df-clab does not define the class
>      abstraction ` { y | ph } ` itself, that is, it does not have the form
>      ` |- { y | ph } = ... ` that a standard definition should have (for a good...

And ´this does not look bad in HTML:
![grafik](https://github.com/user-attachments/assets/41a2faa4-6b2f-4f70-b156-b2caa1526bcb)

Therefore, I have no objections against this change, although I think the previous version was also acceptable:
![grafik](https://github.com/user-attachments/assets/cf931999-4376-4b0f-9abb-55f98d2aaf79)

Reply:
You're right: I noticed too that "semantically", using "..." as a math symbol in the sense of an ellipsis is not correct, but as you write, it's better typographically, and it's only in comments, where the meaning is clear.

In the present case, you're right that there is no big change between both versions.  In more complex formulas, as a general rule I try to avoid "switching from text-mode to math-mode" back and forth, e.g.,
`` ( 3 ` x.` 4 ) ` +. ` ... ` +. ` 2 ``
instead of simply
`` ` ( 3 x. 4 ) +. ... +. 2 ` ``.

Reply:
Maybe we can add this kind of usage to the comment of ` cfz $a class ... $. `, for example

> $( Extend class notation to include the notation for a contiguous finite set of integers.  Read " ` M ... N ` " as "the set of integers from ` M ` to ` N ` inclusive."  Within comments, this symbol is also used as ellipsis, e.g., `` ` ( 3 x. 4 ) +. ... +. 2 ` ``. $)

Reply:
Good idea. Done.